portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -47,8 +49,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
    for (Entity entity : results.asIterable()) {
       long id = entity.getKey().getId();
       String text = (String) entity.getProperty("text");
-
-      Task task2 = new Task(id, text);
+      String email = (String) entity.getProperty("email");
+      Task task2 = new Task(id, text, email);
```

Comment:
I'm getting a build failure (compile error) at this line, and I suspect you forgot to add your updated `Task.java` in your latest commit...? Your Task class constructor now should receive 3 arguments, not 2, and I think the `Task.java` in the repo is not up to date...